pom.xml

```diff
@@ -28,19 +28,6 @@
     <description>Hazelcast In-Memory DataGrid</description>
     <url>http://www.hazelcast.com/</url>
 
-    <modules>
```

Comment:
maven profiles can only extend collections, so in order to allow a profile to build fewer modules we cannot have a list of modules here. Instead, we introduce an active-by-default `default` profile that lists all modules and list fewer modules to be build in `quick` profile.